surveys/2021-annual-survey/questions.md

```diff
@@ -204,6 +204,27 @@ Select all that apply:
 >
 > This question has led to surprising answers in the past that could help the lang team better prioritize features that make interop easier.
 
+### How much do you agree with the following statements
+
+Statements:
+
+- Rust provides a real benefit over other programming languages
+- Rust is complicated to use
+- Rust is risky to use in production
+- Rust makes me less productive
```

Comment:
```suggestion
- Rust makes me more productive
```

I think phrasing this positively is more consistent since I would expect that to be true.